libs/community/langchain_community/tools/pangea/ai_guard.py

```diff
@@ -51,8 +51,9 @@ class PangeaAIGuard(BaseTool):
             ai_guard.invoke("My Name is John Doe and my email is john.doe@email.com.  My credit card number is 5555555555554444.")
     """
 
-    name: str = "Pangea AI Guard Tool"
-    """Name of the tool."""
+    """Name of the tool"""
+    name: str = "pangea-ai-guard-tool"
```

Comment:
Docstring should come after the property actually, so this was fine as it was.

edit: applies to other files of this diff too, plus some cases where it happens to `description`

Reply:
Thanks! It seemed counterintuitive, but so are some other things in Python (still new to me), like the beta designation in versions 🙂 

I found it it now, knowing what I am looking for: https://peps.python.org/pep-0258/#attribute-docstrings

> _It is hard to find a black cat in a dark room when you are not sure it is there_